app_dart/micro_services/results_pipeline/lib/common/response.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
Yes, we need to figure out the mechanism for the common libraries across microservices.

I would be much happier if we could have a versioned dart package that can be imported by each microservice rather than just copying code over. As our number of services increase, and each could have very varied deployment and test cycles, breakages due to incompatible changes could start becoming a real problem.